queries/highlights.scm

```diff
@@ -133,16 +136,20 @@
   "char"
   "bool"
   "string"
+] @type.builtin
+
+[
   "tuple"
   "list"
   "option"
   "result"
   "borrow"
-] @type.builtin
+] @type
```

Comment:
moving `@type` helped give visual clarity between "container" types (`list`, `option`, etc.) and base types such as `char` and `i32`.